workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-julia.md

```diff
@@ -0,0 +1,296 @@
+# From Zero to Binder in Julia!
+
+Sarah Gibson & Oliver Strickson, _The Alan Turing Institute_
+
+[**The Turing Way**](https://github.com/alan-turing-institute/the-turing-way) - making reproducible Data Science "too easy not to do"
+
+Based on Tim Head's _Zero-to-Binder_ workshops which can be found here: <http://bit.ly/zero-to-binder> and <http://bit.ly/zero-to-binder-rise>
+
+To follow these instructions on your own machine, follow this link: **<http://bit.ly/zero-to-binder-julia>**
```

Comment:
That's because 1) the bitly link doesn't exist yet, and 2) when it does, it'll point to the default branch. This way, the link remains consistent even if I update the tutorial. Thanks for checking though :)

Reply:
That's because 1) the bitly link doesn't exist yet, and 2) when it does, it'll point to the default branch. This way, the link remains consistent even if I update the tutorial. Thanks for checking though :)